Makefile

```diff
@@ -47,7 +47,15 @@ vendor:
 .PHONY: vendor
 
 image:
-	podman build -f Dockerfile -t $(LOCAL_IMAGE_TAG) .
+	CI_CONTAINER_VERSION="v1.0" \
```

Comment:
Where do we call this target other then as a developer?  CI has it's own config which calls `make build`.  That does not seem to trace to this target.  What will CPaaS do?  Does this need to be moved to maybe the Dockerfile.in?

Reply:
CI does not use `make build`
CI uses specified Dockerfile path  https://github.com/openshift/release/blob/master/ci-operator/config/openshift/elasticsearch-proxy/openshift-elasticsearch-proxy-release-5.0.yaml#L63-L70

perhaps this change should go into `hack/generate-dockerfile-from-midstream` , we should be able to set all variables except commit_id. does CI has a way to capture it?

Reply:
moved the change to Dockerfile generation

Reply:
> CI does not use `make build`

CI uses `make build` via the Dockerfile https://github.com/openshift/elasticsearch-proxy/blob/master/Dockerfile.in#L19 which will call the default target https://github.com/openshift/elasticsearch-proxy/blob/master/Makefile#L30